Gruntfile.js

```diff
@@ -49,7 +49,8 @@ module.exports = function (grunt) {
             options: {
                 vendor: [
                     'node_modules/jquery/dist/jquery.min.js',
-                    'node_modules/angular/angular.min.js'
+                    'node_modules/angular/angular.min.js',
+                    '<%= paths.js %>/dist/*.js',
```

Comment:
Without this line, sassToJs was not defined within the tests (due to the order of `grunt-contrib-jasmine` including `vendor` -> `helpers` -> `src`). I know it's duplicating the `src` key, but I couldn't get the tests to run otherwise.